homeassistant/helpers/check_config.py

```diff
@@ -125,7 +125,11 @@ def _comp_error(ex: Exception, domain: str, config: ConfigType) -> None:
     for domain in components:
         try:
             integration = await async_get_integration_with_requirements(hass, domain)
-        except (RequirementsNotFound, loader.IntegrationNotFound) as ex:
+        except loader.IntegrationNotFound as ex:
+            if not hass.config.safe_mode:
+                result.add_error(f"Component error: {domain} - {ex}")
```

Comment:
Since we are touching these, let's rename to integration
```suggestion
                result.add_error(f"Integration error: {domain} - {ex}")
```

Reply:
Good call. 👍

Reply:
This is also on a few lines below, but was outside of the "suggestion" range